Library/Homebrew/os/mac/xcode.rb

```diff
@@ -141,9 +141,9 @@ def provides_cvs?
 
       def default_prefix?
         if version < "4.3"
-          %r{^/Developer} === prefix
+          prefix.start_with? "/Developer"
         else
-          %r{^/Applications/Xcode.app} === prefix
+          prefix.start_with? "/Applications/Xcode.app"
```

Comment:
… and this one, too. Sometimes it's nasty having to work with a dynamically typed language. :wink: